docs/guide/data_migration.md

```diff
@@ -514,6 +522,17 @@ final class AssignContentTypeGroupActionDenormalizer extends AbstractActionDenor
 
 ```
 
+Then the action denormalizer needs to be tagged to be recognized by serializer used for migrations.
+
+``` yaml
+services:
+    Ibexa\Platform\Bundle\Migration\Serializer\Denormalizer\ContentType\Action\AssignContentTypeGroupActionDenormalizer:
+        # Autoconfigure set to false will prevent this normalizer from being registered in your application's serializer
```

Comment:
```suggestion
        # When autoconfigure is set to false, this normalizer is not registered in your application's serializer
```

Reply:
True, but this is practically a warning - it's something the user **should not do** (setting `autoconfigure` to `true` in this service).

Usually it should not affect much, although it makes little sense to add Migration normalizers to the default application normalizer.

Reply:
Ok, how about: `Make sure to set autoconfigure to false, otherwise...`?

Reply:
`Make sure to set autoconfigure to false, otherwise this normalizer will be registered in your application's main serializer`?

Reply:
Yep, sounds good